src/db_stats.rs

```diff
@@ -8,22 +8,34 @@ macro_rules! db_stat_name {
 }
 
 pub const IMMUTABLE_MEMTABLE_FLUSHES: &str = db_stat_name!("immutable_memtable_flushes");
+pub const FILTER_FALSE_POSITIVES: &str = db_stat_name!("filter_false_positives");
```

Comment:
nit: WDYT about naming these `bloom_filter` or `bfilter_` so it's clear that it applies to bloom filters if we ever add other kind of filtering stats (like TTL filtering or whatever)